float32 yaw
float32 pitch
---
byte status

